### ISSUE TITLE
Item hex tags are stripped out by the screen-reader, incorrectly assuming they're colour tags

### ISSUE THREAD

**Issue by Gravecat** [Gravecat/Greave#1]
For example, "Skeleton Key {07b}" is just read as "Skeleton Key" on a screen reader. This should be a simple enough fix, but worth putting on the to-do list.

**Comment by Gravecat** [comment]
Looks like the issue is in core/core.cpp, in Core::message(). There's a regex filter to remove anything in { } braces, assuming that'd be colour tags. That's likely what's also removing the hex tags from items in the inventory.

**Comment by Gravecat** [comment]
Nevermind, looks like this one was actually an easy fix, even for a regex noob like me. :D
